code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -263,6 +263,8 @@
 // MOVE SPEED //
 ////////////////
 #define ADD_SLOWDOWN(__value) if(!ignoreslow || __value < 0) . += __value
+#define SLOWDOWN_INCREMENT 0.5
+#define SLOWDOWN_MULTIPLIER (1 / SLOWDOWN_INCREMENT)
 
 /datum/species/proc/movement_delay(mob/living/carbon/human/H)
 	. = 0	//We start at 0.
```

Comment:
could you make the `if(has_gravity(H))` check below be an early return

Reply:
done